lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -11,7 +11,6 @@
 
 from capa.tests.response_xml_factory import StringResponseXMLFactory
 from courseware.courses import get_course_by_id  # pyline: disable=import-error
-from courseware.field_overrides import OverrideFieldData  # pylint: disable=import-error
 from courseware.tests.factories import StudentModuleFactory  # pylint: disable=import-error
 from courseware.tests.helpers import LoginEnrollmentTestCase  # pylint: disable=import-error
```

Comment:
Are you sure these disables are needed? At L13 you don't even use the right syntax.